content/800-data-platform/100-accelerate/600-faq.mdx

```diff
@@ -0,0 +1,78 @@
+---
+title: 'FAQ'
+metaTitle: 'FAQ'
+metaDescription: 'Frequently asked questions about Accelerate.'
+tocDepth: 3
+toc: true
+---
+
+<TopBlock>Below are frequently asked questions about Accelerate.</TopBlock>
+
+## Why do I sometimes see unexpected cache behaviour?
+
+Accelerate's cache performs best when it observes a higher load from a project. Many cache operations, such as committing data to cache and refreshing stale data, happen asynchronously. When benchmarking Accelerate, we recommend doing so with loops or a load testing approach. This will mimic higher load scenarios better and reduce outliers from low frequency operations.
+
+Prisma operations are sent to Accelerate over HTTP. As a result, the first request to Accelerate must establish an HTTP handshake and may have additional latency as a result. We're exploring ways to reduce this initial request latency in the future.
+
+## What happens to my Scale plan subscription if I switch to Accelerate?
+
+While Accelerate is in Preview, you do not need a subscription or a monthly plan. With Accelerate, you can use the same Data Proxy [connection pooling features](/data-platform/accelerate/getting-started#no-cache-strategy-to-only-use-connection-pool) on top of the Accelerate global cache.
+
+In addition, you can get 6 months of free Accelerate usage when you opt-in for the credits program and enter your payment information (debit or credit card).
+
+## What is the pricing of Accelerate?
+
+When we officially launch Accelerate and make it generally available, we will provide pricing details.
+
+## VSCode does not recognize the `$extends` method
+
+If you add the Prisma Client extension for Accelerate to an existing project that is currently open in VSCode, the editor might not immediately recognize the `$extends` method.
+
+This might be an issue with the TypeScript server not yet recognizing the regenerated Prisma Client. To resolve this, you need to restart TypeScript.
+
+1. In VSCode, open the Command Palette. You can do so when you press F1 or select **View** > **Command Palette**.
+2. Enter `typescript` and select and run the **TypeScript: Restart TS server** command.
+
+VSCode should now recognize the `$extends` method.
```

Comment:
```suggestion
## VS Code does not recognize the `$extends` method

If you add the Prisma Client extension for Accelerate to an existing project that is currently open in VS Code, the editor might not immediately recognize the `$extends` method.

This might be an issue with the TypeScript server not yet recognizing the regenerated Prisma Client. To resolve this, you need to restart TypeScript.

1. In VS Code, open the Command Palette. You can do so when you press F1 or select **View** > **Command Palette**.
2. Enter `typescript` and select and run the **TypeScript: Restart TS server** command.

VS Code should now recognize the `$extends` method.
```